pkg/monitor/api/drop.go

```diff
@@ -100,6 +100,7 @@ var errors = map[uint8]string{
 	202: "Host datapath not ready",
 	203: "Endpoint policy program not available",
 	204: "No Egress IP configured",
+	205: "Encryption requires invalid SPI",
```

Comment:
I think this new drop reason should be documented in the troubleshooting section of the IPsec guide.

Reply:
ack, done now.